app/assets/stylesheets/protocols.scss

```diff
@@ -62,3 +62,7 @@
 /// Dashboard ///
 /////////////////
 
+#epic_rights_error_alert {
```

Comment:
This is cosmetic but you should be able to use bootstrap padding and margin classes instead of using a stylesheet

Reply:
Unfortunately not really. Those only have preset sizing, none of which are correct for this situation. So unless I create a whole new entry into the variables for sizing for just this one thing, I have to use custom css (at least for the padding)